### DISCUSSION TITLE
Inductance L2 Motherboard

### DISCUSSION THREAD

**Discussion by JasiuBEL** [PE5PVB/SI4684-DAB-Receiver#2] [Q&A]
Hey,
What should be the value of the inductance?
Currently it is L2,0603,1,BEAD,
Motherboard

**Comment by PE5PVB** [comment]
Just use a ferrit inductor.

**Reply by JasiuBEL** [reply]
Thank you.

I am having trouble completing my order for JLCPCB ;-)
The most important U3 SI4684-A10-GM is missing from stock. ;-/

**Reply by PE5PVB** [reply]
Yup, You have to order these parts first and they will be added to your private stock at JLC PCB. That's how I did it.

**Reply by JasiuBEL** [reply]
Sjef,
So, you ordered the chip and sent it to them for them to mount on the board?
I ask because this is the first time I'm ordering with assembly. So far, I've ordered just the boards without mounting. I noticed in your video that you ordered with assembly and I would like to try it too. Another experience. ;-)

**Reply by PE5PVB** [reply]
See: https://jlcpcb.com/help/article/60-how-to-use-jlcpcb-global-sourcing-parts-service

**Reply by JasiuBEL** [reply]
Thank you, you have helped me a lot.
